app.py

```diff
@@ -0,0 +1,54 @@
+"""
+This is the main code of my program, Guess the number !!!
+"""
+import random
+import time
+
+
+# Functions
+def get_guess_Player():
```

Comment:
```suggestion
def get_guess_player():
```

lowercase to be consistent `with_snake_case`

Reply:
agree